docs/l10n.rst

```diff
@@ -329,6 +329,39 @@ the template with the context. You can use the ``ftl_files`` argument with this
     def about_view(request):
         render(request, 'about.html', {'name': 'Duder'}, ftl_files='mozorg/about')
 
+Fluent File Configuration
+-------------------------
+
+In order for a Fluent file to be extracted through automation and sent out for localization,
+it must first be configured to go through one or more distinct pipelines. This is controlled
+via a set of configuration files:
+
+- `Vendor`_, locales translated by an agency, and paid for by Marketing (locales covered by staff are also included in this group).
+- `Pontoon`_, locales translated by Mozilla contributors.
+- `Special templates`_, for locales with dedicated templates that don't go through the localization process.
```

Comment:
I'm still confused about this. Why do we need configuration for this if there won't be a fluent file for these templates? Is it just for the case that we have a fallback template and would want to exclude locales that are already hard coded to avoid superfluous work? Maybe we can add some more information about these at some point.

Reply:
Yeah, I asked @Pike about this in https://github.com/mozilla/bedrock/pull/8092#pullrequestreview-313368378

I'm still not quite sure as to the real world use case. I guess we could perhaps use this for a pseudo-localization locale. Something that used Fluent files (e.g for testing purposes), but doesn't leave bedrock?

Reply:
I would encourage the use of different terminology here. "Vendor" will include staff locales too, meaning that for some locales it won't be translated by an agency. Maybe "external" is the better designation?

Reply:
Yes, this is about avoiding un-used work. @gueroJeff told me that staff can do that inside BWX, though. They can decline a particular item of work.

Probably need to gather some experience to see if that's working well with our flow.

Reply:
@gueroJeff I’m using vendor here as a term because that’s what the configuration file itself is called. I’m happy to rename it, but note that the docs above do call out that vendor includes staff locales also.

Reply:
Gotcha, thanks for clarifying. If other places make that clarifying reference, I think this should be fine.